node/impl/full.go

```diff
@@ -37,4 +40,40 @@ func (n *FullNodeAPI) CreateBackup(ctx context.Context, fpath string) error {
 	return backup(n.DS, fpath)
 }
 
+func (n *FullNodeAPI) NodeStatus(ctx context.Context) (status api.NodeStatus, err error) {
+	curTs, err := n.ChainHead(ctx)
+	if err != nil {
+		return status, err
+	}
+
+	status.SyncStatus.Epoch = uint64(curTs.Height())
+	timestamp := time.Unix(int64(curTs.MinTimestamp()), 0)
+	delta := time.Since(timestamp).Seconds()
+	status.SyncStatus.Behind = uint64(delta / 30)
+
+	pis, err := n.NetPeers(ctx)
+	if err != nil {
+		return status, err
+	}
```

Comment:
This obtains the currently _connected_ peers, which is a subset of peers with scores, right? Can we add a comment. It's not immediately obvious why we obtain peers separately, when `NetPubsubScores` already returns peers.

Reply:
right, NetPubScores is all peers within the retention window; will add comment.

Reply:
added comment.